docs/organizations/using-a-coding-standard.md

```diff
@@ -12,7 +12,7 @@ The organization coding standard helps you ensure that Codacy analyzes multiple
 When you customize the tools or code patterns of a repository that follows the coding standard, Codacy warns you that the repository will stop following the coding standard and asks for your confirmation.
 
 !!! important
-    The coding standard enables or disables any tools that are using configuration files on the repositories following the coding standard, but these tools will keep using configuration files instead of the code pattern settings of the coding standard.
+    [Tool configuration files](../repositories-configure/configuring-code-patterns.md#using-your-own-tool-configuration-files) override coding standards.
```

Comment:
Is this the gist? The original gives some more details, but I found it hard to follow. Let me know if I lost something!

Reply:
Your sentence is correct but omits the detail that the coding standard will still affect the on/off status of the tools using configuration files.

I'm also concerned that it loses a bit of context that may be important for users reading this page because in your version the information is stated "from the point of view" of tool configuration files, which is a big jump away from the discussion around the concept of the coding standard.

My suggestion is that you clarify the information by splitting it into separate sentences:

- The coding standard turns tools with configuration files on and off
- However, the configuration files take precedence over the code patterns defined on the coding standard

Reply:
Keeping it singular for now:
```suggestion
    The coding standard turns tools with configuration files on and off. Tool configuration files, however, take precedence over the code patterns defined on the coding standard.
```